components-java-sdk/src/main/java/io/dapr/components/wrappers/RequestStreamToFluxAdaptor.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dapr.components.wrappers;
+
+import io.dapr.components.domain.bindings.ReadRequest;
+import io.dapr.components.domain.pubsub.PublishRequest;
+import io.grpc.stub.StreamObserver;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Sinks;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+import static reactor.core.publisher.Sinks.EmitFailureHandler.FAIL_FAST;
+
+/**
+ * Creates a StreamObserver that adapts and feeds its received requests to a Flux.
+ *
+ * <p>It also does peaks the very first response and stores it aside becasue both
```

Comment:
Nit: `peeks` and `because`

Reply:
Fixed on c2118d079950bbca990d1cb54e59826edb1d9ac9